helm/reana/values.yaml

```diff
@@ -53,7 +53,7 @@ components:
     enabled: true
   reana_server:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-server:0.6.0-35-gafd0a7f
+    image: reanahub/reana-server:0.6.0-51-gcf52449
```

Comment:
To be amended, new build including https://github.com/reanahub/reana-server/pull/249.